### ISSUE TITLE
Обнаружены циклические ссылки

### ISSUE THREAD

**Issue by mrDSide** [oscript-library/opm#86]
Любой вызов 'opm' даёт такой результат:

> $ opm help list
> {Модуль /usr/share/oscript/lib/opm/src/opm.os / Ошибка в строке: 0 / Ошибка загрузки библиотеки /usr/share/oscript/lib/opm/src. Обнаружены циклические зависимости.
> -> /usr/share/oscript/lib/cmdline
>   -> /usr/share/oscript/lib/logos
>     -> /usr/share/oscript/lib/opm/src
> }

> $ oscript
> 1Script Execution Engine. Version 1.0.18.30

**Comment by nixel2007** [comment]
@EvilBeaver @dmpas в пол линукс так и не пересобирали на сайте пакет? Это оттуда ноги

**Comment by dmpas** [comment]
да, оттуда ноги...

**Comment by nixel2007** [comment]
@dmpas у тебя вроде какой-то отдельные бранч с фиксом был? Может его вмержить просто? И черрипикнуть в 17ый релиз

**Comment by dmpas** [comment]
@nixel2007 нет, там в настройках среды сборки дело. Простым коммитом в движок дело не решить.

**Comment by nixel2007** [comment]
@dmpas могу чем помочь? Ключи доступа есть.

**Comment by EvilBeaver** [comment]
Закрываю же?
